assets/aml-benchmark/components/src/dataset_downloader/loaders/math.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Why are we hosting these scripts in our component code? I thought they would be input to the component? Shouldn't they be a data-asset in the azureml registry?

How do we invoke these scripts?

Reply:
With having these loader scripts in the component, people can directly use them by using the custom loader file_name, for example `mmlu`, in the component's argument `dataset_name`. Also, this eliminates the need to host these scripts in the registry.

Reply:
Just a thought- is it possible that the component the script is in this repo (not in component) and we provide a link to it as URL_FILE input? That way, it becomes clear to the user how download is working?

Reply:
yes, this will be better. I will add these loaders outside the component's code, and the url of the script can be sent in the `script_path` argument.

Reply:
Please add a test for it (if not already done). We can make sure it works. Thanks!

Reply:
test is already added.